model/src/main/kotlin/OrtResult.kt

```diff
@@ -446,11 +446,14 @@ data class OrtResult(
      * [omitResolved] and remove violations below the [minSeverity].
      */
     @JsonIgnore
-    fun getRuleViolations(omitResolved: Boolean = false, minSeverity: Severity? = null): List<RuleViolation> {
+    fun getRuleViolations(
```

Comment:
The commit message does not refer to `getRuleViolations()`, which makes it sound as if `minSeverity` is a general model property. I propose:

    refactor(model)!: Align severity filtering in `getRuleViolations()`

    Avoid dealing with a nullable `minSeverity` in the same was as in
    `getIssues()`.